plutus-tx-plugin/test/Plugin/Errors/literalCaseBs.plc.golden

```diff
@@ -1 +1,3 @@
-Error: Unsupported feature: Use of == from the Haskell Eq typeclass
\ No newline at end of file
+Error: Reference to a name which is not a local, a builtin, or an external INLINABLE function: Variable PlutusTx.Builtins.Internal.$fEqBuiltinByteString
```

Comment:
I guess we should do the trick for `==` too, but we also probably want to keep the old fallback error which was helpful.